### DISCUSSION TITLE
How to rotate focused container?

### DISCUSSION THREAD

**Discussion by ok-nick** [nikitabobko/AeroSpace#736] [questions-and-answers] [answered]
Hi, I was wondering how to rotate the focused container, e.g. horizontal->vertical, vertical->horizontal. Initially, I expected "layout tiles horizontal vertical" to do this, but it seems it rotates the root container instead? In i3, I believe it was "layout toggle split" for this behavior.

Here's an example of what I'm looking for:
```
h_tiles
├── window 1
└── v_tiles
    ├── window 2 (focused)
    └── window 3
```
When "rotate focused container" is applied:
```
h_tiles
├── window 1
└── h_tiles <--- notice how this changes
    ├── window 2 (focused)
    └── window 3
```

**Comment by nikitabobko** [comment, marked as answer]
Please look for normalizations in the docs. You need to disable one of them. Though I recommend doing that only if you are a hardcore i3 user, disabling normalizations leads to weird layouts

**Reply by ok-nick** [reply]
`enable-normalization-opposite-orientation-for-nested-containers` - this is the setting I was looking for
